ContactDetailsApi.Tests/V1/E2ETests/Steps/CreateContactSteps.cs

```diff
@@ -28,23 +28,6 @@ public class CreateContactSteps : BaseSteps
         public CreateContactSteps(HttpClient httpClient) : base(httpClient)
         { }
 
-        private async Task<HttpResponseMessage> CallApi(string id, bool? includeHistoric = null)
```

Comment:
Are these two methods not used anywhere?

Reply:
Not any more. 
The test had been unnecessarily re-loading the created record to verify it via the API, but I refactored it so that it reloaded it by calling the DB directly.

Reply:
Ah okay, Makes sense